app/routers/event.py

```diff
@@ -172,6 +182,31 @@ async def create_new_event(
     )
 
 
+def process_image(
+    img: bytes,
+    event_id: int,
+    img_height: int = IMAGE_HEIGHT,
+) -> str:
+    """Resized and saves picture without exif (to avoid malicious date))
+    according to required height and keep aspect ratio"""
+    try:
+        image = Image.open(io.BytesIO(img))
+    except IOError:
+        error_message = "The uploaded file is not a valid image"
+        logger.exception(error_message)
+        return
+    width, height = image.size
+    height_to_req_height = img_height / float(height)
+    new_width = int(float(width) * float(height_to_req_height))
+    resized = image.resize((new_width, img_height), Image.ANTIALIAS)
+    file_name = f"{event_id}{PICTURE_EXTENSION}"
+    image_data = list(resized.getdata())
+    image_without_exif = Image.new(resized.mode, resized.size)
+    image_without_exif.putdata(image_data)
+    image_without_exif.save(f"{EVENT_IMAGES_PATH}/{file_name}")
```

Comment:
I think it might be useful to randomize the name to prevent people overriding other people images (if some event have two `wedding_invite.png`, we should allow it)

Reply:
The name is actually the event's id , not the name of the uploaded picture. Therefore there will be only one picture associated to the event, and it is easier to associate it with the event.